### ISSUE TITLE
Server crash when try to second carpet on stairs

### ISSUE THREAD

**Issue by GordonHollow** [EndlesNights/CarpetStairsMod#12]
Server crashes when anyone trying second carpet on stairs to cover face. Minecraft 1.14.4, Carpet Stairs Mod mc1.14 v1.3.2, forge-1.14.4-28.2.4

Here the crashlog:

```
Description: Exception in server tick loop

java.lang.NoSuchMethodError: net.minecraft.block.Block.func_200291_n()Lnet/minecraft/util/text/ITextComponent;
	at com.endlesnights.carpetstairsmod.PlaceHandlerCarpetStairs.getDyeColorBlock(PlaceHandlerCarpetStairs.java:139) ~[?:1.3.2] {re:classloading}
	at com.endlesnights.carpetstairsmod.PlaceHandlerCarpetStairs.placeCarpet(PlaceHandlerCarpetStairs.java:66) ~[?:1.3.2] {re:classloading}
	at com.endlesnights.carpetstairsmod.PlaceHandlerCarpetStairs.onBlockEntityPlace(PlaceHandlerCarpetStairs.java:37) ~[?:1.3.2] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_134_PlaceHandlerCarpetStairs_onBlockEntityPlace_RightClickBlock.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:80) ~[eventbus-1.0.0-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:258) ~[eventbus-1.0.0-service.jar:?] {}
	at net.minecraftforge.common.ForgeHooks.onRightClickBlock(ForgeHooks.java:785) ~[?:?] {re:classloading}
	at net.minecraft.server.management.PlayerInteractionManager.func_219441_a(PlayerInteractionManager.java:313) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.network.play.ServerPlayNetHandler.func_184337_a(ServerPlayNetHandler.java:847) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches ServerPlayNetHandler Transformer}
	at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:36) ~[?:?] {re:classloading}
	at net.minecraft.network.play.client.CPlayerTryUseItemOnBlockPacket.func_148833_a(SourceFile:10) ~[?:?] {re:classloading}
	at net.minecraft.network.PacketThreadUtil.func_225383_a(SourceFile:21) ~[?:?] {re:classloading}
	at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?] {re:classloading}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(SourceFile:144) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?] {re:classloading}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(SourceFile:118) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:701) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:695) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(SourceFile:103) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:680) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:628) [?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_241] {}
```

**Comment by EndlesNights** [comment]
I had fixed this issue on the 1.15 branch a while back but I neglected the 1.14 branch. My bad. I've updated 1.14 branch to be in line with 1.15  (minus support for the Extended Mushroom mod), but includes the server crash fix, some render fixes as well as the slim carpet stairs 
Hope this works out well for you. Let me know if there are any other issues.

**Comment by GordonHollow** [comment]
Yes, all works fine!
